bot/commands/command_manager.go

```diff
@@ -153,6 +154,18 @@ func (cm *CommandManager) OnMessage(message discordgo.Message) (err error) {
 
 	commandObject, exists := cm.Commands[commandTrigger]
 	if !exists {
+		newdist := 100
```

Comment:
It would be good to start with a lower threshold, the distance between `carl johnson` and `carter jenkins` is only 9 so this would end up suggesting commands for every message sent.

Reply:
A threshold of only 2 or 1 would suffice. That way, if you type `vertify` it would suggest `verify` but it wouldn't work for `vertigo` or `verified`.